samples/apps/copilot-chat-app/webapp/src/components/chat/chat-list/ChatList.tsx

```diff
@@ -60,7 +59,7 @@ export const ChatList: FC = () => {
                                 leaf
                                 style={
                                     id === selectedId
-                                        ? { background: 'var(--colorNeutralBackground1Selected)' }
+                                        ? { background: '#FFFFFF' }
```

Comment:
This will override the background color if the theme is Dark

Reply:
Instead:

```suggestion
                                        ? { background: tokens.colorNeutralBackground1Selected }
```

and import { tokens } from '@fluentui/react-components';